paymentsheet/src/test/java/com/stripe/android/lpmfoundations/paymentmethod/PaymentMethodMetadataTest.kt

```diff
@@ -59,7 +59,10 @@ internal class PaymentMethodMetadataTest {
 
     @Test
     fun `filterSupportedPaymentMethods removes unsupported paymentMethodTypes`() {
-        val metadata = PaymentMethodMetadataFactory.create()
+        val stripeIntent = PaymentIntentFixtures.PI_REQUIRES_PAYMENT_METHOD.copy(
```

Comment:
How come this change is necessary?

Reply:
So that we indeed have an unsupported payment method type. If we use the default returned from the factory, we get the two supported types `card` and `link`.